src/main/java/ulcrs/GsonFactory.java

```diff
@@ -0,0 +1,16 @@
+package ulcrs;
+
+import com.google.gson.Gson;
+import com.google.gson.GsonBuilder;
+
+public class GsonFactory {
```

Comment:
I like the idea of a Gson Factory, though see my comment in the CourseController.

Reply:
Got it. Changed Gson factory to provide gson and exposeOnlyGson.

Reply:
I think this approach is good, it's clean and intuitive